client/js/components/App.react.jsx

```diff
@@ -1,26 +1,60 @@
+import _ from 'lodash';
 import React from 'react';
+import turf from 'turf';
 
 import api from '../utils/api';
+import utils from '../utils';
 import ParkMap from './ParkMap.jsx';
-import ParkList from './ParkList.jsx';
+import ParksList from './ParksList.jsx';
+import ParksMap from './ParksMap.jsx';
```

Comment:
Lol, I'm terrible at naming.

ParksMap = the map with many parks on the homepage

ParkMap = the map of a single park with all its features and such